web/src/components/AssertionForm/Fields/AttributeField.tsx

```diff
@@ -0,0 +1,58 @@
+import {Form, Select} from 'antd';
+import {FormListFieldData} from 'antd/lib/form/FormList';
+import {uniqBy} from 'lodash';
+import React, {ReactElement, useMemo, useState} from 'react';
+import {TSpanFlatAttribute} from '../../../types/Span.types';
+import * as S from '../AssertionForm.styled';
+import {OtelReference} from '../hooks/useGetOTELSemanticConvertionAttributesInfo';
+import {dropDownRenderComponent} from './dropDownRenderComponent';
+import {referencePicker} from './referencePicker';
+
+interface IProps {
+  attributeList: TSpanFlatAttribute[];
+  reference: OtelReference;
+  field: Pick<FormListFieldData, never>;
+  name: number;
+}
+
+export const AttributeField = ({field, name, reference, attributeList}: IProps): ReactElement => {
+  const [hoveredKey, setHoveredKey] = useState<string | undefined>(undefined);
+  return (
+    <Form.Item
+      {...field}
+      name={[name, 'attribute']}
+      style={{margin: 0}}
+      rules={[{required: true, message: 'Attribute is required'}]}
+      data-cy="assertion-check-attribute"
+      id="assertion-check-attribute"
+    >
+      <S.Select
+        style={{margin: 0, display: 'flex'}}
+        placeholder="Attribute"
+        showSearch
+        dropdownRender={dropDownRenderComponent(hoveredKey, reference)}
+        dropdownStyle={{minWidth: 550, maxWidth: 550}}
```

Comment:
Isn't this something we could add as part of the styled-components?

Reply:
no, dropdown is not a child of the select